regulations/tasks.py

```diff
@@ -110,8 +116,8 @@ def html_to_pdf(html):
         path = tempfile.mkdtemp()
         html_path = os.path.join(path, 'document.html')
         pdf_path = os.path.join(path, 'document.pdf')
-        with open(html_path, 'w') as fp:
-            fp.write(html)
+        with open(html_path, 'wb') as fp:
+            fp.write(ensure_utf8(html))
```

Comment:
Huh, I tried that earlier and ran into encoding errors, but this snippet works fine in python 2 and 3. I wonder what I had wrong before. Anyway, fixing this now.